### PR TITLE
Add a few more terms

### PR DESCRIPTION
**Pull request by giant-atomic-monster**
A few more terms for `choice()` to pick from

### CODE REVIEW

File: TitleCreator.py [hunk 1]
```diff
@@ -5,11 +5,12 @@
 
 def ofprefix():
     l = ['Viceroy', 'Commandant', 'Grand Poo-Bah', 'Archon', 'Duke', 'Chancellor', 'President', 'Marquis',
-            'Earl', 'Director', 'Chair', 'Head', 'Senior Director', 'Vice President', 'Dark Lord']
+            'Earl', 'Director', 'Chair', 'Head', 'Senior Director', 'Vice President', 'Dark Lord',
```
Comment (gabinante) [inline TitleCreator.py:9]: Can we remove 'Administrator' due to https://github.com/gabinante/TitleCreator/issues/5 ? Or feel free to make a PR addressing this issue :)

File: TitleCreator.py [hunk 2]
```diff
@@ -5,11 +5,12 @@
...
+            'Dread Lord', 'Czar', 'Emperor', 'Baron', 'Guru', 'Administrator']
     return choice(l)
 
 def prefix():
-    l = ['Principal', 'Chief', 'Head', 'Lead', 'Senior', 'Master']
+    l = ['Principal', 'Chief', 'Head', 'Lead', 'Senior', 'Master', 'Arch', 'Grand', 'Premier']
```
Comment (gabinante) [inline TitleCreator.py]: Arch/Grand don't seem to flow here (arch systems engineer?) but I like Premier. Can you remove the other two from this PR so I can merge it?

File: TitleCreator.py [hunk 3]
```diff
@@ -20,7 +21,7 @@ def job():
 def ofpostfix():
     l = ['Engineering',
             'Management', 'Development', 'Deployment', 'Technical Training', 'Operations', 'Architecture',
-            'Infrastructure', 'Technology', 'Administration', 'Thought Leadership']
+            'Infrastructure', 'Technology', 'Administration', 'Thought Leadership', 'Shiny Things']
```
Comment (gabinante) [inline TitleCreator.py:25]: Genius.

**Review by gabinante — CHANGES_REQUESTED** [review]
minor changes, issue 5

**Comment by gabinante** [comment]
Also, feel free to rebase the readme and add yourself to contributors - otherwise I'll get around to it after your changes are merged.

File: TitleCreator.py (hunk 1 above)
Comment (giant-atomic-monster) [reply TitleCreator.py:9]: Done.

**Review by gabinante — APPROVED** [review]

**Pull request merged**